### DISCUSSION TITLE
Improving sethome icon selection menu

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#9151] [Enhancement]
Hello, can you add way to "search" the icon that can be set for the home menu, please? Currently it's a bit difficult to find the wanted icon, as you need to go through bunch of the pages

So like search button, where you can type, for example "Spyglass" & it will find spyglass item that can be set as the icon

**Comment by t347549** [comment]
Hi, what do you think about drag-and-drop icon replacement? It's something already seen in /warps and many other GUI-based plugins.
https://github.com/Zrips/CMI/discussions/9176

**Comment by CloudeLecaw** [comment]
well why not extant it a bit with headdatabase support the plugin has hover xxxxxx heads :3
